landingPage/assets/css/styles.css

```diff
@@ -0,0 +1,10 @@
+#user-posts {
+  display: inline-flex;
+  flex-wrap: nowrap;
+  margin-left: 10px;
```

Comment:
Instead of using this line of code, for the #user-posts class, add the 'ml-2' class to it in the DOM.
For example <section id='user-posts' class='col-12 ml-2'>